src/components/cylc/GraphNode.vue

```diff
@@ -18,43 +18,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
   <g class="c-graph-node">
     <!-- the task icon -->
-    <symbol :id="nodeID" viewBox="-40 -40 140 140">
```

Comment:
It would appear these symbol/use objects can no longer be reliably measured :shrug:.

Symbols are kinda like "sprites", usually used for repeated elements in the document for efficiency reasons which don't apply for our single-use use case.

We were using symbols because these were more easily clickable. Click events landed on the parent "use" element and they worked through the svgPanZoom layer with its default click event configuration.

Reply:
Here's a screenshot showing the GraphNode component selected in the FF devtools showing the incorrect dimensions returned before this change:

![graph-error](https://github.com/cylc/cylc-ui/assets/16705946/b21b3213-2f54-47ee-9dd2-04129a0f59a4)